src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -140,7 +140,8 @@ public static int Run(string[] args)
                                    new { Name = "Web", isMsBuild = true }, 
                                    new { Name = "Lib", isMsBuild = true },
                                    new { Name = "Mstest", isMsBuild = true },
-                                   new { Name = "Xunittest", isMsBuild = true }
+                                   new { Name = "Xunittest", isMsBuild = true },
```

Comment:
Just to confirm, there was nothing that was needed to allow this template to run on a non-empty folder?